app/code/Aspire/Module/Block/Adminhtml/MultiSelect.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Aspire\Customer\Block\Adminhtml;
+
+class MultiSelect extends \Magento\Config\Block\System\Config\Form\Field
+{
+
+    /**
+     * Retrieve Element HTML fragment
+     *
+     * @param AbstractElement $element
+     * @return string
+     */
+    protected function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractElement $element)
+    {
+        $script = " <script>
+                require([
```

Comment:
@kartikaspire Implement the Javascript in Magento way.

Reply:
This file is not required